tests/devices/qutrit_mixed/test_qutrit_mixed_apply_operation.py

```diff
@@ -0,0 +1,426 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for create_initial_state in devices/qutrit_mixed/apply_operation."""
+
+import os
+import pytest
+import numpy as np
+from scipy.stats import unitary_group
+import pennylane as qml
+from pennylane import math
+from pennylane.operation import Channel
+from pennylane.devices.qutrit_mixed.apply_operation import (
+    apply_operation_einsum,
+    apply_operation,
+)
+
+ml_frameworks_list = [
+    "numpy",
+    pytest.param("autograd", marks=pytest.mark.autograd),
+    pytest.param("jax", marks=pytest.mark.jax),
+    pytest.param("torch", marks=pytest.mark.torch),
+    pytest.param("tensorflow", marks=pytest.mark.tf),
+]
+
+methods = [apply_operation_einsum, apply_operation]  # TODO add tensordot back when added
+broadcasting_methods = [apply_operation_einsum, apply_operation]
+subspaces = [(0, 1), (0, 2), (1, 2)]
+
+
+def load_data(num_qutrits, batched):
```

Comment:
I'm not personally the biggest fan of this strategy. Storing test data in a separate file makes sense, but `.npy` files are binaries that are reasonably hard to inspect. I don't know what any of the states stored in the files look like. Though I'm not opposed to a new approach and also want to get other opinions on this.

Reply:
I think it is better to use a `.py` file, for the reasons you mention.

Reply:
Sounds good I will store them in a `.py` file.

Reply:
its now in `conftest.py` with seed to make the readability as good as possible

Reply:
@glassnotes let me know if you are happy with this solution

Reply:
I'm quite happy with this solution :)

Reply:
thanks @mudit2812